t/setup_test_config.pm

```diff
@@ -0,0 +1,91 @@
+package setup_test_config;
+=head1 NAME
+
+setup_test_config
+
+=cut
+
+=head1 DESCRIPTION
+
+setup_test_config
+Setups the configuration for the testing environment
+
+=cut
+
+use strict;
+use warnings;
+
+
+our $PFCONFIG_TEST_PID_FILE;
+
+BEGIN {
+    use test_paths;
+    use pfconfig::manager;
+    use pfconfig::constants;
+    use File::Spec::Functions qw(catfile);
+    use File::Slurp qw(read_file);
+
+    use File::Path qw(remove_tree);
+    remove_tree('/tmp/chi');
+
+    $PFCONFIG_TEST_PID_FILE = "/usr/local/pf/var/run/pfconfig-test.pid";
+    `/usr/local/pf/sbin/pfconfig -n pfconfig-test -s $pfconfig::constants::SOCKET_PATH -p $PFCONFIG_TEST_PID_FILE -c $pfconfig::constants::CONFIG_FILE_PATH -d`;
```

Comment:
It would also make sense for pfconfig process to use test_paths also.
perl -I/usr/local/pf/t -M test_paths /usr/local/pf/sbin/pfconfig -n pfconfig-test -p $PFCONFIG_TEST_PID_FILE -d